src/lib.rs

```diff
@@ -2463,21 +2479,18 @@ impl Connection {
         Ok(())
     }
 
-    /// Drops the initial keys and recovery state.
-    fn drop_initial_state(&mut self) {
-        if self.pkt_num_spaces[packet::EPOCH_INITIAL]
-            .crypto_open
-            .is_none()
-        {
+    /// Drops the keys and recovery state for the given epoch.
+    fn drop_epoch_state(&mut self, epoch: usize) {
```

Comment:
```suggestion
    fn drop_epoch_state(&mut self, epoch: packet::Epoch) {
```

We have a typedef for Epoch, so lets use it

Reply:
Oh, yeah, good point, I forgot about that.